Add .linn files for NN customer training session

maxwell-ashurst opened the pull request:
Presenting a customer training session at photonics west. Adding some .linn files to the github for some of the examples we will be using so that customers don't need to wait for training during the session.

bnizette-li commented:
Thanks @maxwell-ashurst, can you please just name the folder "models" and add a README explaining that these are pre-trained models that correspond to the example notebooks. Also, these don't all seem to precisely correspond to example notebooks: Is Signal_Classifier as a model Signal_ID as a notebook? What's the Averager one? Please set all those up to match, then I'll happily merge

maxwell-ashurst commented:
I've removed the Averager model, renamed the Signal_Classifier to SigID to match the notebook and included a ReadMe. Let me know if I need to do anything more to get this merged. Not all the examples are included here, as I only need the three .linn files in this merge request for the training session
